python_modules/dagster/dagster_tests/core_tests/definitions_tests/decorators_tests/test_op.py

```diff
@@ -841,55 +841,38 @@ def test_output_generic_correct_inner_type():
     def the_op_not_using_output() -> Output[int]:
         return 42
 
-    result = execute_op_in_graph(the_op_not_using_output)
-    assert result.success
-
-    assert the_op_not_using_output() == 42
-
-    @op
-    def the_op_annotation_not_using_output() -> int:
-        return Output(42)
-
-    result = execute_op_in_graph(the_op_annotation_not_using_output)
-    assert result.success
-
-    result = the_op_annotation_not_using_output()
-    assert isinstance(result, Output)
-    assert result.value == 42
-
-
-def test_output_generic_type_mismatches():
-    @op
-    def the_op_annotation_type_mismatch() -> int:
-        return Output("foo")
```

Comment:
why were these tests removed?

Reply:
They were testing a bunch of type mismatch behavior that is now irrelevant since we now error in a much broader case.